### PR TITLE
Input

### PR DESCRIPTION
**Pull request by pariyar07**

### CODE REVIEW

**Review by yogesh- — APPROVED** [review]
Looks good to merge

File: pages/input/input.css [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+::after {
+    box-sizing: border-box;
+    margin: 0px;
+    padding: 0px;
+}
+
+body {
+    font-family: Poppins, sans-serif;
+    background-color: #000000;
+    color: rgb(255, 255, 255);
```
Comment (syed-shaida-hussain) [inline pages/input/input.css:14]: try to use variable for color here as well

File: pages/input/input.css [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+    flex-direction: column;
+    background-color: #171717;
+    min-height: 300px;
+    width: 500px;
+    padding: 1rem 1.2rem;
+    border-radius: 10px;
+    align-items: flex-end;
+    margin-bottom: 5rem;
+}
+
```
Comment (syed-shaida-hussain) [inline pages/input/input.css:34]: Don't repeat yourself . Make flex utility class and then use it everywhere.

File: pages/input/input.css [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+.textarea{
+    height: 250px;
+    width: 100%;
+    border-radius: 5px;
+    padding: 1rem 1.2rem;
+    outline: none;
+    background-color: rgb(225, 225, 225);
```
Comment (syed-shaida-hussain) [inline pages/input/input.css:41]: use variable for colors

File: pages/input/input.css [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+    border: none;
+    padding: 10px;
+    font-family: inherit;
+    font-size: 16px;
+    display: block;
+    width: 100%;
+    border-radius: 4px;
+    cursor: pointer;
+}
+
```
Comment (syed-shaida-hussain) [inline pages/input/input.css:158]: try to use relative units instead of pixels.

File: pages/input/input.js [hunk 5]
```diff
@@ -0,0 +1,63 @@
+const container = document.querySelector('.container');
```
Comment (syed-shaida-hussain) [inline pages/input/input.js:1]: i think component library was to be made using HTML and CSS only.

**Review by syed-shaida-hussain — CHANGES_REQUESTED** [review]
Try using variables for colors and use relative units.

File: pages/input/input.css [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+body {
+    font-family: Poppins, sans-serif;
+    background-color: #000000;
+    color: rgb(255, 255, 255);
+    min-height: 100vh;
+    margin-top: 5rem; 
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
```
Comment (vaibhav18matere) [inline pages/input/input.css:20]: can make separate class for it ex. `d-center`

File: pages/input/input.html [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+                <input type="text" id="username" placeholder="Username">
+                <i id="suc" class="fas fa-check"></i>
+                <i id="err" class="fas fa-exclamation-circle"></i>
+                <small>Message</small>
+            </div>
+            <div class="form-control">
+                <label for="email">Email</label>
+                <input type="email" id="email" placeholder="Email">
+                <i id="suc" class="fas fa-check"></i>
+                <i id="err" class="fas fa-exclamation-circle"></i>
```
Comment (vaibhav18matere) [inline pages/input/input.html:36]: give proper ID names ex.` success` `error` to make it more clear

**Pull request merged**